javalib/src/main/scala/java/util/zip/InflaterOutputStream.scala

```diff
@@ -0,0 +1,103 @@
+// ported from android luni 2e317a02b5a8f9b319488ab9311521e8b4f87a0a
```

Comment:
I believe we should add a license info mentioning the usage of Android Luni source code into `License.md` file (similarly to usage RE2 and others, at the end of the file)

Reply:
@WojciechMazur good point, done.